server/swagger.yaml

```diff
@@ -162,21 +163,33 @@ paths:
               properties:
                 username:
                   type: string
+                  format: username
+                  example: alex
```

Comment:
What does this example mean and for what usage?

Reply:
It will display on the UI as the default input, so we could save some time typing it.